config/projects.yml

```diff
@@ -1,4 +1,63 @@
 admin-bundle:
+  custom_gitignore_part: |
+    # clean up some non bower ready package
+    Resources/public/vendor/admin-lte/bootstrap
```

Comment:
Do we need a full list. Can't we just uningnore `Resources/public/vendor/admin-lte/` (and the other root dirs)?

Reply:
I am not sure WHY this was done, I just want to keep the current one. Improvments of the gitignore itself could be done in another PR, but I think when we merge https://github.com/sonata-project/SonataAdminBundle/pull/5461 it s obsolete anyway